src/marketplace/ListingsView.js

```diff
@@ -4,21 +4,34 @@ import {getTankComponent, verifyComponent} from '../armory/GetInventoryInfo.js';
 import Cookies from 'universal-cookie';
 import type {ComponentType} from '../armory/ComponentType.js';
 
+
+interface itemForSaleObject {
```

Comment:
err, correct me if I'm wrong, but this seems like exactly the job for a class, no? Usually you would use an interface if you have a bunch of classes that all share a couple methods but are different. All of these are just items for sale, right?